src/metabase/email/messages.clj

```diff
@@ -212,15 +212,17 @@
           :logoFooter   true}
          (random-quote-context)))
 
+(defn- render-message-body
+  [message-template message-context images]
+  (vec (cons {:type "text/html; charset=utf-8" :content (stencil/render-file message-template message-context)}
+             (mapv write-image-content images))))
```

Comment:
I think if you're calling `vec` on the result of the call to `cons` there's no real need to do `mapv` here

Reply:
(It doesn't make a difference though so I wouldn't bother changing it)